gtsam/gtsam.i

```diff
@@ -2940,6 +2941,7 @@ class ShonanAveragingParameters3 {
   void setOptimalityThreshold(double value);
   double getOptimalityThreshold() const;
   void setAnchor(size_t index, const gtsam::Rot3& value);
+  pair<size_t, Rot3> getAnchor();
```

Comment:
gtsam::Rot3

Reply:
Good catch. Weird that the CI checks pass !?